.changeset/angry-days-swim.md

```diff
@@ -0,0 +1,31 @@
+---
+'@commercetools-frontend/application-shell-connectors': patch
+---
+
+There is a new React hook which you can use to access the Project Images rewrite configuration [see documentation](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#project-image-settings).
+
+You would use it like this:
+
+```js
+function MyComponent() {
+  const { isLoading, imageRegex } = useProjectExtensionImageRegex();
+
+  if (isLoading) return <LoadingSpinner />;
+
+  return (
+    <div>
+      <h1>Project images regex: {imageRegex}</h1>
+    </div>
+  );
+}
+
+function MyApp() {
+  return (
+    <ProjectExtensionProviderForImageRegex>
+      <MyComponent />
+    </ProjectExtensionProviderForImageRegex>
+  );
+}
+```
+
+Both GetProjectExtensionImageRegex component and withProjectExtensionImageRegex still exists for backwards compatibility but have been marked as deprecated.
```

Comment:
There's still this formatting missing:

```suggestion
Both `GetProjectExtensionImageRegex` component and `withProjectExtensionImageRegex` still exists for backwards compatibility but have been marked as deprecated.
```

Reply:
Thank you:https://github.com/commercetools/merchant-center-application-kit/pull/2639/commits/eedc5fb61cc0c725ea07443e19f0360b0a27d0a6